client/src/main/java/org/schemarepo/client/Avro1124RESTRepositoryClient.java

```diff
@@ -93,7 +93,7 @@ public Subject lookup(String subject) {
   public Iterable<Subject> subjects() {
     ArrayList<Subject> subjectList = new ArrayList<Subject>();
     try {
-      String subjects = webResource.get(String.class);
+      String subjects = webResource.accept(MediaType.TEXT_PLAIN_TYPE).get(String.class);
```

Comment:
Well... if we have separate HTML endpoints that will essentially duplicate most of the `RESTRepository` methods (with different `@Path` annotations) *or* it will be simpler to have a separate `HTMLRepository` which will be living parallel to `RESTRepository`. Is that what you have in mind?